node/pkg/governor/governor_db.go

```diff
@@ -18,6 +18,9 @@ func (gov *ChainGovernor) loadFromDB() error {
 	return gov.loadFromDBAlreadyLocked()
 }
 
+// Loads transfers and pending data from the database and modifies the corresponding fields in the ChainGovernor.
```

Comment:
I think the doc generator expects the function prologues to start with the function name.